.github/workflows/docs.yml

```diff
@@ -0,0 +1,39 @@
+name: Run Documentation Validation
+
+on:
+  workflow_dispatch:
+  push:
+  pull_request:
```

Comment:
is this valid yaml?

Reply:
Should be valid, but it might be better to structure it as a list